Vagrantfile

```diff
@@ -41,8 +41,33 @@ else
 end
 
 # Multiple public_network mappings need at least 1.7.4
-# Custom triggers stop working in 2.1.0 so that is max for now
-Vagrant.require_version '>= 1.7.4', '< 2.1.0'
+Vagrant.require_version '>= 1.7.4'
+
+# Try to use the vagrant-triggers plugin when using a vagrant version below 2.1.0 or
+# if the user has self chosen to do so by setting the VAGRANT_USE_VAGRANT_TRIGGERS
+# environment variable.
+use_triggers_plugin = false
+triggers_plugin_installed = Vagrant.has_plugin? 'vagrant-triggers'
+if Gem::Version.new(Vagrant::VERSION) >= Gem::Version.new('2.1.0')
+  # Vagrant core triggers are available, but the user can also opt-in to
+  # use the plugin if it is installed
+  if triggers_plugin_installed
+    use_triggers_plugin = true
+    if ENV['VAGRANT_USE_VAGRANT_TRIGGERS'].nil?
+      # Exit here because starting Vagrant will not work if VAGRANT_USE_VAGRANT_TRIGGERS is
+      # not defined but at the same time the vagrant-triggers plugin is installed.
+      exit 1
```

Comment:
Should here be some puts to tell the user why 'vagrant up' stopped?

Reply:
Added an error message and instructions to uninstall the vagrant-triggers plugin.